src/test_typing_extensions.py

```diff
@@ -4763,34 +4820,34 @@ class Base(TypedDict, closed=True):
         self.assertEqual(Base.__extra_items__, ReadOnly[Union[str, None]])
         self.assertTrue(Base.__closed__)
 
-        class Child(Base):
+        class Child(Base, closed=True):
```

Comment:
```suggestion
        class Child(Base):
```

We should test the case of a child with a closed parent (1/2)

Reply:
This test tests the legacy `__extra_items__: T` syntax, so I'd rather leave it alone, and instead add a new test working as you suggest.